lib/loader.js

```diff
@@ -57,10 +57,11 @@ module.exports = function (content) {
 
   var bubleOptions = hasBuble && options.buble ? '?' + JSON.stringify(options.buble) : ''
   var defaultLoaders = {
-    html: templateCompilerPath + '?id=' + moduleId,
+    html: '',
     css: (isServer ? '' : styleLoaderPath + '!') + 'css-loader' + (needCssSourceMap ? '?sourceMap' : ''),
     js: hasBuble ? ('buble-loader' + bubleOptions) : hasBabel ? 'babel-loader' : ''
   }
+  var templateCompilerLoaderCommand = templateCompilerPath + '?id=' + moduleId
```

Comment:
If defaultLoaders.html is left as-is, then for the default case, `var loader = loaders[lang]` on line 140 grabs that string and the new code on 151-153 below also add the same string, resulting in `template-compiler?id=abc!template-compiler?id=abc` for the final loader string.  

My rationale for pulling it out into a separate variable was to make the 2 cases similar:
- In the default case, the format of the template is HTML, which doesn't need a loader to translate it to HTML (`''`)
- In the case where a user specifies a lang, it's entry will be the loader specified in the config.

In both cases, `var loader = loaders[lang]` will grab the loader necessary to translate the input language to HTML, and then we can prepend `template-compiler?id=abc` to it to handle compiling the HTML spit out by the loader.

Having said that, if you prefer a different style, let me know.

Thanks!